src/kirby/components/badge/badge.component.tns.scss

```
@@ -0,0 +1,48 @@
+@import "../../scss/native-imports";
+
+.kirby-badge Label { 
```

The styling needs to be applied to the host element, to allow for icons with a badge background color. See https://github.com/kirbydesign/designsystem/blob/master/src/kirby/components/button/button.component.tns.global.scss for how it's done

OR

simply style the `<AbsoluteLayout>` container:
```suggestion
.kirby-badge { 
```